### PR TITLE
Improvements to api integration with vurl

### PR DESCRIPTION
**Pull request by SL-Pirate**
- Improved data saved to vurl
- added endpoint to retrieve data from vurl

### CODE REVIEW

File: controllers/send_emails/signature_link.js [hunk 1]
```diff
@@ -46,13 +46,19 @@ const signatureLinkEmail = async (req, res) => {
         });
 
         const emailPromises = [];
+        console.log(associativeArray);
```
Comment (visald) [inline controllers/send_emails/signature_link.js:49]: remove this console.log

File: routes/v1/application.js [hunk 2]
```diff
@@ -95,6 +95,14 @@ router.post("/email/send", routesMiddleware, async function (req, res, next) {
   await sendEmails(req, res);
 });
 
+router.get("/applicant", routesMiddleware, async function (req, res, next) {
+  const data = await req.cache.json.get("signature_los") || [];
```
Comment (visald) [inline routes/v1/application.js:99]: use optional chaining here

Comment (SL-Pirate) [reply routes/v1/application.js]: Changed
Please check again

Comment (visald) [reply routes/v1/application.js]: just add chaining - if u don't know how to chain, check the example I provided below.

req?.cache?.json?.get("signature_los")

**Pull request merged**